.eslintrc.js

```diff
@@ -2,26 +2,27 @@ module.exports = {
   extends: ["prettier", "eslint:recommended", "plugin:react/recommended"],
   plugins: ["prettier", "react", "jest", "react-hooks"],
   rules: {
+    "no-prototype-builtins": "warn",
```

Comment:
I changed this to warn, because existing code was failing the check:

https://github.com/trufflesuite/ganache-ui/pull/5071/files#diff-c9854b8ced114e2e270fade1360f54409639eafbbe87118a9f14209462c6bce4L6